### PR TITLE
add iot topic

### PR DESCRIPTION
**Pull request by StokicDusan**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/master/CONTRIBUTING.md>.
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/master/docs>

> It's on the `topics-todo.md` list

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/iot/index.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+--
+aliases: IoT, Internet-of-things
```
Comment (kenyonj) [inline topics/iot/index.md:2]: These are case insensitive so we don’t want a duplicate of the topic name here. 
```suggestion
aliases: Internet-of-things
```

File: topics/iot/index.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+display_name: Internet of things
+related: cloud, embedded, devices
+short_description: The iot is a network of devices which exchanging data with other devices and systems over the internet.
```
Comment (kenyonj) [inline topics/iot/index.md:5]:
```suggestion
short_description: The iot is a network of devices that exchange data with other devices and systems over the internet.
```

**Review by kenyonj — DISMISSED** [review]
Just a couple comments and then we can run CI

**Comment by StokicDusan** [comment]
Committed changes suggested by @kenyonj

File: topics/iot/index.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+--
```
Comment (octokatherine) [inline topics/iot/index.md:1]:
```suggestion
---
```

**Review by octokatherine — CHANGES_REQUESTED** [review]
👋  @StokicDusan, there is a syntax error causing CI failures, I left a suggested change!

**Review by hodies528 — COMMENTED** [review]
完美的诠释

**Review by MikeMcQuaid — APPROVED** [review]
Thanks so much for your first contribution! Without people like you submitting PRs we couldn't run this project. You rock, @StokicDusan!

**Pull request merged**